Client/game_sa/CStreamingSA.h

```diff
@@ -31,4 +29,8 @@ class CStreamingSA : public CStreaming
     void RequestSpecialModel(DWORD model, const char* szTexture, DWORD channel);
 
     CStreamingInfo* GetStreamingInfoFromModelId(ushort id);
+
+private:
+    static CStreamingInfo (&s_aInfoForModel)[26316];
+    CStreamingInfo (&CStreamingSA::ms_aInfoForModel)[26316] = *(CStreamingInfo(*)[26316])0x8E4CC0;
```

Comment:
It's better to move this line to the **cpp**. H headers are just for declaration. We don't want to initialize the variable in the every translation unit.

Reply:
Line 35 I mean